### DISCUSSION TITLE
👋 I'm looking for volunteers to submit servers!

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1941] [Q&A]
In short, I've been working on a new https://speedtest-tracker.dev site and I need volunteers to submit a list of their local Ookla servers. 

I'm testing a new feature that will list and validate known servers (on the site and via an API) and it needs to be community driven. 

If you're interested just post "me" below 👇 and when I'm ready I'll be reaching out.

**Comment by svenvg93** [comment]
Me

**Comment by jetkins** [comment]
Me

**Comment by publi0** [comment]
Me

**Comment by seh2000** [comment]
Me too

**Comment by Skyeci73** [comment]
Im in!

Me
